applications_tests/lethe-particles/periodic_boundary_load_balancing.prm

```diff
@@ -46,9 +46,7 @@ end
 #---------------------------------------------------
 
 subsection lagrangian physical properties
-  set gx                       = 0.0
-  set gy                       = -1
-  set gz                       = 0.0
+  set g                        = 0.0, -1, 0.0
```

Comment:
Why this gravity?

Reply:
Values of test don't need to be realistic. Sometime, (let's speak for myself) we need to tuned parameter in order to reproduce a specific bug or to reduce simulation time